### ISSUE TITLE
Merge into react-native-geolocation?

### ISSUE THREAD

**Issue by cristianoccazinsp** [Agontuk/react-native-geolocation-service#150]
I gotta say that the Android version of this library is much better than the one from the original library. Wouldn't it be a good idea to merge this there? Any reason not to?

**Comment by SCasarotto** [comment]
I would also like to know why not just merge into the community lib

**Comment by cristianoccazinsp** [comment]
With recent changes, this is no longer valid.  This library now has an implementation for both iOS and Android, which imho is far superior than the community one.

**Comment by SCasarotto** [comment]
Just seems strange to me to create a lib with the primary purpose to fix a problem in another lib instead of attempting to PR into it. Feels like it results in more fracture in the community than stability. 

If this library serves more than:

> This library is created in an attempt to fix the location timeout issue on android with the react-native's current implementation of Geolocation API.

I suggest the README be updated to reflect that and highlight the differences / improvements.

**Comment by cristianoccazinsp** [comment]
I agree with the suggestion! It started as an android improvement, but ended up with an improved API / implementation.  Perhaps @Agontuk can review this and update the readme.

**Comment by Agontuk** [comment]
Not possible yet, community geolocation library is still needed for developers who build apps for Chinese region since google play service is not available in China.
